### PR TITLE
Niaz erdoolatov

### PR DESCRIPTION
**Pull request by neojan84**
little changes

### CODE REVIEW

**Comment by neojan84** [comment]
2 files added

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # test-cases
 Test cases repositories
-hello
+hello everybody 1
```
Comment (Pokhilenko) [inline README.md:3]: nice counter)

File: Niaz/testCheck.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+CREATE PROCEDURE solution()
+    SELECT id, IF ( given_answer IS NULL, 'no answer', IF ( given_answer =correct_answer, 'correct', 'incorrect') ) AS checks
+    FROM answers
+    ORDER BY id;
```
Comment (Pokhilenko) [inline Niaz/testCheck.sql:4]: Seems like `END` is missing here

Comment (Pokhilenko) [inline Niaz/testCheck.sql:4]: Fix indentations all over the files, please

File: Niaz/suspectInvestigation2.sql [hunk 3]
```diff
@@ -0,0 +1,7 @@
+CREATE PROCEDURE solution()
+BEGIN
+select id, name, surname from suspect
+where height <= 170
+OR lower(name) not like 'b%'
+OR lower(surname) not like 'Gre_n';
```
Comment (Pokhilenko) [inline Niaz/suspectInvestigation2.sql:6]: please, unify commands' case: make them all in lower or all in upper case

File: Niaz/expressionsVerification.sql [hunk 4]
```diff
@@ -0,0 +1,11 @@
+CREATE PROCEDURE solution()
```
Comment (Pokhilenko) [inline Niaz/expressionsVerification.sql:1]: Move all `.sql` files to `sql-assignment` folder under the `Niaz` folder

**Review by Pokhilenko — CHANGES_REQUESTED** [review]
Here are some changes you need to make before merging this PR to the main branch

File: Niaz/testCheck.sql (hunk 2 above)
Comment (Pokhilenko) [reply Niaz/testCheck.sql:4]: Figured out, that it's okay for this assignment, no need to add `END`

Comment (neojan84) [reply Niaz/testCheck.sql]: fixed where needed

File: Niaz/suspectInvestigation2.sql (hunk 3 above)
Comment (neojan84) [reply Niaz/suspectInvestigation2.sql]: made it in upper case

File: Niaz/expressionsVerification.sql (hunk 4 above)
Comment (neojan84) [reply Niaz/expressionsVerification.sql]: Created and moved everything to a new folder.

**Review by Pokhilenko — APPROVED** [review]

**Pull request merged**